MapboxNavigation/RouteMapViewController.swift

```diff
@@ -358,25 +302,6 @@ class RouteMapViewController: UIViewController {
         }
     }
 
-    private func showStatus(title: String, withSpinner spin: Bool = false, for time: TimeInterval, animated: Bool = true, interactive: Bool = false) {
```

Comment:
Did anything change here or was this simply moved?

Reply:
just moved